### PR TITLE
features/30751-Scrollable-Documentation

### PR DESCRIPTION
**Pull request by DevanshiGandhi**
Documentation for Scrollable component is ready for code review.

### CODE REVIEW

File: docs/src/examples/ScrollableExample/ScrollableExampleFirst.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+            under the Terms of Service include various products and services to help you create and manage
+            a retail store, whether an online store (“Online Services”), a physical retail store
+            (“POS Services”), or both. Any new features or tools which are added to the current Service
+            shall be also subject to the Terms of Service. You can review the current version of the Terms of
+            Service at any time at https://www.shopify.com/legal/terms.</p>
+        </Scrollable>
+        <br/>
+
+        2. Scrollable Container with prop:
+        <Scrollable shadow vertical horizontal
```
Comment (mkulinski) [inline docs/src/examples/ScrollableExample/ScrollableExampleFirst.tsx]: Each prop should be on a different line.

File: example/src/examples/ScrollableExample/ScrollableExampleFirst.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
```
Comment (mkulinski) [inline example/src/examples/ScrollableExample/ScrollableExampleFirst.tsx]: Why is this file in the example folder?

File: example/src/reducers/componentState/ScrollableState.ts [hunk 3]
```diff
@@ -0,0 +1,37 @@
+import { IDocument } from '../../Types';
```
Comment (mkulinski) [inline example/src/reducers/componentState/ScrollableState.ts]: Why is this file here?

File: example/src/examples/ScrollableExample/ScrollableExampleFirst.tsx (hunk 2 above)
Comment (DevanshiGandhi) [reply example/src/examples/ScrollableExample/ScrollableExampleFirst.tsx]: Yes, you are right, I am removing that file from example folder now.

File: example/src/reducers/componentState/ScrollableState.ts (hunk 3 above)
Comment (DevanshiGandhi) [reply example/src/reducers/componentState/ScrollableState.ts]: Yes, you are right, I am removing that file from example folder now.
